tests/src/JIT/Directed/Misc/gettype/gettypetypeofmatrix.csproj

```diff
@@ -22,8 +22,7 @@
     </CodeAnalysisDependentAssemblyPaths>
   </ItemGroup>
   <PropertyGroup>
-    <!-- Set to 'Full' if the Debug? column is marked in the spreadsheet. Leave blank otherwise. -->
-    <DebugType></DebugType>
+    <DebugType>None</DebugType>
```

Comment:
@tmat  - we may have a bug here. CSC wants DiaSymReader on Unix.

As I understand both  debug:full  and debug:pdbonly   are basically the same thing and duplication is for historical reasons.
In either case the format of debug information should fallback to “portable” on Unix and not require DiaSymReader.

Our check for "IsUnix" is basically `Path.DirectorySeparatorChar == '/'`, I hope that is the case in your environment.



Basically whichever way you do it, you will